lib/perl/Genome/Model/Tools/Capture/FormatSnvs.t

```diff
@@ -4,10 +4,48 @@ use strict;
 use warnings;
 
 use above 'Genome';
-use Test::More tests => 1;
+use Test::More;
+use Genome::Utility::Test;
```

Comment:
The `use Genome::Utility::Test qw(compare_ok);` should be sufficient to cover both uses of the module, and this one can be removed.